aws/table_aws_backup_report_plan.go

```diff
@@ -0,0 +1,196 @@
+package aws
+
+import (
+	"context"
+	// "fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/backup"
+
+	// "github.com/aws/aws-sdk-go-v2/service/backup/types"
+
+	backupv1 "github.com/aws/aws-sdk-go/service/backup"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBackupReportPlan(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_backup_report_plan",
+		Description: "AWS Backup Report Plan",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValueException"}),
```

Comment:
@madhushreeray30, should we not ignore the not found error (`ResourceNotFoundException`) here? 
I can see in the [doc](https://docs.aws.amazon.com/aws-backup/latest/devguide/API_DescribeReportPlan.html) The API throws `ResourceNotFoundException` if the resource is not available.

Reply:
Also, please add the resource not found test case in the integration test.